techniques/css/C29.html

```diff
@@ -241,27 +237,27 @@ <h5>In altColors4.css</h5>
 			</section>
 			<section id="javascriptComponents">
 			
-				<h4>The JavaScript components</h4>
+				<h4>JavaScript コンポーネント</h4>
 			
 <pre xml:space="preserve"><code class="language-javascript">function changeColors (newCSS){
   document.getElementById('currentCSS').href = newCSS; 
 }</code></pre>
 			
-			<p class="working-example">A working example of this code, <a href="../../working-examples/script-swap-css/">Using a JavaScript control to apply a different external CSS file</a>, is available.</p>
+			<p class="working-example">このコードの実装例 <a href="https://www.w3.org/WAI/WCAG22/working-examples/script-swap-css/">Using a JavaScript control to apply a different external CSS file</a> が利用可能である。</p>
```

Comment:
working example の訳を、他のテクニックと合わせて「動作例」としました。
```suggestion
			<p class="working-example">このコードの動作例 <a href="https://www.w3.org/WAI/WCAG22/working-examples/script-swap-css/">Using a JavaScript control to apply a different external CSS file</a> が利用可能である。</p>
```